src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/DetailsViewModelMapper.cs

```diff
@@ -82,11 +85,13 @@ public async Task<DetailsViewModel> Map(DetailsRequest source)
                     HasPendingChangeOfPartyRequest = pendingChangeOfPartyRequest != null,
                     PendingChangeOfPartyRequestWithParty = pendingChangeOfPartyRequest?.WithParty,
                     HasApprovedChangeOfPartyRequest = approvedChangeOfPartyRequest != null,
+                    HasPendingChangeOfProviderRequest = pendingChangeOfProviderRequest != null,
                     EncodedNewApprenticeshipId = approvedChangeOfPartyRequest?.NewApprenticeshipId != null
                         ? _encodingService.Encode(approvedChangeOfPartyRequest.NewApprenticeshipId.Value,
                             EncodingType.ApprenticeshipId)
                         : null,
                     IsContinuation = data.Apprenticeship.IsContinuation && data.Apprenticeship.PreviousProviderId == source.ProviderId,
+                    HasContinuation = data.Apprenticeship.HasContinuation,
```

Comment:
Is it possible to use the Approved flag (Change of party request) like on task CON-2585 to hide the link if it is a previous provider and there is a completed CoP reqeuest. Then we don't need the das-commitment changes.